llvm/include/llvm/Support/SYCLRTShared.h

```diff
@@ -0,0 +1,29 @@
+//=- SYCLRTShared.h - Shared definition between llvm tools and SYCL runtime -=//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#ifndef LLVM_SUPPORT_SYCLRTSHARED_H
+#define LLVM_SUPPORT_SYCLRTSHARED_H
+/// Device binary image property set names recognized by the SYCL runtime.
+/// Name must be consistent with
+/// PropertySetRegistry::SYCL_SPECIALIZATION_CONSTANTS defined in
+/// PropertySetIO.h
+#define PI_PROPERTY_SET_SPEC_CONST_MAP "SYCL/specialization constants"
+#define PI_PROPERTY_SET_DEVICELIB_REQ_MASK "SYCL/devicelib req mask"
```

Comment:
```suggestion
#define SYCL_PROPERTY_SET_SPEC_CONST_MAP "SYCL/specialization constants"
#define SYCL_PROPERTY_SET_DEVICELIB_REQ_MASK "SYCL/devicelib req mask"
```

Reply:
Thanks for moving it here. Now, it should be removed from PropertySetIO.[cpp|h] too, and all sources (post-link tool and SYCL runtime) which use that should include this header.

Reply:
Done.